cmd/podmanV2/containers/run.go

```diff
@@ -108,13 +108,13 @@ func run(cmd *cobra.Command, args []string) error {
 	}
 	runOpts.Spec = s
 	report, err := registry.ContainerEngine().ContainerRun(registry.GetContext(), runOpts)
+	registry.SetExitCode(report.ExitCode)
```

Comment:
Can we get a comment on why this has to be before the return so we don't accidentally undo this later?

Reply:
Added